docs/template_plugin/tests/functional/transformations/template_transformations_test.cpp

```diff
@@ -18,7 +18,7 @@
 using namespace testing;
 
 // ! [transformation:test]
-TEST(TransformationTests, TemplateTest) {
+TEST(TransformationTests, DISABLED_TemplateTest) {
```

Comment:
why this test is disabled? Does not it pass even in master?

Reply:
It dose not work in master